CHANGELOG.md

```diff
@@ -3,14 +3,35 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.0dev - [date]
+## [1.0.0](https://github.com/nf-core/fastqrepair/releases/tag/1.0.0) - Catanzaro YellowRed [25/01/2025]
 
 Initial release of nf-core/fastqrepair, created with the [nf-core](https://nf-co.re/) template.
 
-### `Added`
+### Credits
```

Comment:
Fine for first release, but for second release please use the conventional headers and for future reelases: https://github.com/nf-core/tools/blob/main/nf_core/pipeline-template/CHANGELOG.md